src/utils/decorators/input-label/input-label.js

```diff
@@ -116,9 +116,9 @@ const InputLabel = ComposedComponent => class Component extends ComposedComponen
      * A string representing a help message
      *
      * @property
-     * @type {String}
+     * @type {Node}
      */
-    fieldHelp: PropTypes.string,
+    fieldHelp: PropTypes.node,
```

Comment:
Will this still support strings?

Reply:
Node will support anything: https://reactjs.org/docs/typechecking-with-proptypes.html#proptypes